src/main/kotlin/app/revanced/cli/command/ListPatchesCommand.kt

```diff
@@ -118,6 +115,6 @@ internal object ListPatchesCommand : Runnable {
         val filtered =
             packageName?.let { patches.filter { (_, patch) -> patch.filterCompatiblePackages(it) } } ?: patches
 
-        if (filtered.isNotEmpty()) logger.info(filtered.joinToString("\n\n") { it.buildString() })
+        if (filtered.isNotEmpty()) println(filtered.joinToString("\n\n") { it.buildString() })
```

Comment:
The logger should be used instead of println.

Reply:
> The logger should be used instead of println.

Is it possible to suppress the "INFO:" thing that it outputs?